pingpong/test_lti_platforms.py

```
@@ -0,0 +1,278 @@
+}
+
+
+def _lxp_register_request(provider_id: int = 0) -> LTIRegisterRequest:
+    return LTIRegisterRequest(
+        name="PingPong",
+        admin_name="Admin",
+        admin_email="admin@example.com",
+        provider_id=provider_id,
+        sso_field=None,
+        openid_configuration="https://platform.example.com/.well-known/openid",
+        registration_token="token",
```

Ruff (0.15.10)

[error] Possible hardcoded password assigned to argument: "registration_token"

(S106)